pipeline-model-definition/pom.xml

```diff
@@ -120,7 +120,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>git</artifactId>
-      <scope>test</scope>
+      <optional>true</optional>
```

Comment:
I don't know? My guess is there's something else in the dependency tree that's getting a newer version when git is a non-test dependency.